velox/functions/prestosql/aggregates/tests/SumTest.cpp

```diff
@@ -224,20 +224,21 @@ TEST_F(SumTest, sumFloat) {
 }
 
 TEST_F(SumTest, sumDouble) {
-  // Run the test multiple times to ease reproducing the issue:
-  // https://github.com/facebookincubator/velox/issues/2198
   for (int iter = 0; iter < 3; ++iter) {
     SCOPED_TRACE(fmt::format("test iterations: {}", iter));
-    auto rowType = ROW({"c0", "c1"}, {REAL(), DOUBLE()});
+    auto rowType = ROW({"c0", "c1", "c2"}, {REAL(), REAL(), INTEGER()});
```

Comment:
Test name may need updating as it says sumDouble. Would it make sense to add a separate test for float?

Reply:
I change to sumDoubleAndFloat and it might be good to test them together so we have mixed aggregation types. Thanls!